nservicebus/azure-service-bus/securing-connection-strings.md

```diff
@@ -6,56 +6,56 @@ tags:
+In certain scenarios this could lead to insecure behavior and result in a leak of such connection strings, for example when messages are exchanged with untrusted parties (native outgoing integration, messages export, etc) or when body content is added to [log files](/nservicebus/logging/) which are then shared.
 
-To prevent this kind of accidental leaking, Versions 7 and above can map a logical namespace name to a namespace connection string. By default the connection string is still being passed around, but that behavior can be changed to using the `UseNamespaceNamesInsteadOfConnectionStrings()` API Setting.
+To prevent this kind of accidental leaking, Versions 7 and above can map an alias to a namespace connection string. By default the connection strings are still passed around. To override the default behavior use the `UseNamespaceNamesInsteadOfConnectionStrings()` configuraion API setting.
 
-snippet: enable_use_namespace_name_instead_of_connection_string
+snippet: enable_use_namespace_alias_instead_of_connection_string
 
-If this setting is enabled, the resulting `ReplyTo` header will look like this:
+When this setting is enabled, `ReplyTo` header will no longer contain raw connection string and will be structured as following
 
 ```no-highlight
-[queue name]@[namespacename]
+[queue name]@[alias]
 ```
 
 
 ## Configuration
 
-Mapping between a namespace name and the corresponding connection string, to perform cross namespace routing, is done as follows:
+To perform a cross-namespace routing, connection string has to be mapped to a corresponding alias. For example:
 
 snippet: namespace_routing_registration
 
-Note: All endpoints in a system need to be configured with the same namespace names and connection string information.
+Note: All endpoints in a system need to be configured with the same namespace aliases and connection string information.
 
-For a detailed explanation about all ways to configure namespace mappings for namespace routing and namespace partitioning, see [Multiple Namespaces Support](multiple-namespaces-support.md).
+For a detailed explanation on configuring namespace mappings for namespace routing and partitioning, see [Multiple Namespaces Support](multiple-namespaces-support.md).
 
 
 ## Backward compatibility
 
 include: asb-credential-warning
 
-Internally the transport (Version 7) uses the namespace name to refer to namespaces. Even when using the `ConnectionString(string connectionString)` method on the configuration API directly, as shown below, it will cause the transport to add a mapping between a namespace name `default` and the provided connection string internally.
+Internally the transport (Version 7 an above) uses namespace aliass to refer to namespaces. Even when using the `ConnectionString(string connectionString)` method on the configuration API directly, as shown below, it will cause the transport to add a mapping between a namespace alias `default` and the provided connection string internally.
```

Comment:
should be `Versions 7 and above`. See version lang: https://github.com/particular/docs.particular.net#version-language

typo in `aliass`